docs/references/api.md

```diff
@@ -334,3 +337,199 @@ No exceptions raised by this endpoint.
 ### CLI interaction
 #### Calling from the CLI:
 This endpoint currently has no CLI support. Only the Python-based version is available.
+
+
+## u.pro.security.status.livepatch_cves.v1
+Lists Livepatch patches for the current running kernel.
+
+### Args
+This endpoint takes no arguments.
+
+### Python API interaction
+#### Calling from Python code
+```python
+from uaclient.api.u.pro.security.status.livepatch_cves.v1 import livepatch_cves
+
+result = livepatch_cves()
+```
+
+#### Expected return object:
+`uaclient.api.u.pro.security.status.livepatch_cves.v1.LivepatchCVEsResult`
+
+|Field Name|Type|Description|
+|-|-|-|
+|fixed_cves|list(LivepatchCVEObject)|List of Livepatch patches for the given system|
+
+`uaclient.api.u.pro.security.status.livepatch_cves.v1.LivepatchCVEObject`
+|Field Name|Type|Description|
+|-|-|-|
+|name|str|Name (ID) of the CVE|
+|patched|bool|LIvepatch has patched the CVE|
+
+
+### Raised Exceptions
+No exceptions raised by this endpoint.
+
+
+### CLI interaction
+#### Calling from the CLI:
+```bash
+pro api u.pro.security.status.livepatch_cves.v1
+```
+
+#### Expected attributes in JSON structure
+```json
+{
+    "fixed_cves":[
+        {
+            "name": "<CVE Name>",
+            "patched": True
```

Comment:
minor nit: booleans are lowercase in json
```suggestion
            "patched": true
```

Reply:
fixed